test/test_helper.exs

```diff
@@ -1,7 +1,3 @@
-unless System.get_env("NR_INT_TEST") do
-  {:ok, _} = NewRelic.EnabledSupervisor.start_link(enabled: true)
```

Comment:
We start the supervisor individually in each test that needs it instead of globally, so that we can test it starting explicitly